vehicle/mkvi/software/imu/icm20948/icm20948.c

```diff
@@ -0,0 +1,71 @@
+#include <stdint.h>
```

Comment:
nit: it seems that Amit started a convention of throwing common drives (ex ltc6811) into the vehicle/common dir, probably worth throwing this in there (unless you anticipate it could change, causing previous firmware versions to break)

Reply:
The general API should stay the same. I've thrown it in `vehicle/common`.